implementations.tex

```diff
@@ -14,6 +14,10 @@ \section{Abstract Command Based}
 Memory is accessed using the Abstract Access Memory command or through System
 Bus Access.
 
+Since this implementation doesn't rely on the hart executing any instructions,
+it could allow a debugger to collect information from the hart even if it is
+otherwise completely hung.
```

Comment:
I don't really know what "hung" means here. "Unable to execute instructions" makes more sense, though it's a bit redundant with the above.